src/main/java/uk/gov/hmcts/reform/orgrolemapping/helper/AccessTypesBuilder.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.reform.orgrolemapping.helper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.reform.orgrolemapping.controller.advice.exception.BadRequestException;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.AccessType;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.AccessTypeJurisdiction;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.AccessTypesResponse;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.OrganisationProfile;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.OrganisationProfileAccessType;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.OrganisationProfileJurisdiction;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.RestructuredAccessTypes;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class AccessTypesBuilder {
+
+    public static RestructuredAccessTypes restructureCcdAccessTypes(AccessTypesResponse accessTypesResponse) {
```

Comment:
I dont like the use of these static classes in this manner. Its better if you set up this builder as a proper service and get rid of the static methods
That way the code is more maintainable and provides a single responsibility